### PR TITLE
docs(ja): translate `vscode` into Japanese

### PR DESCRIPTION
**Pull request by sakimyto**
## Summary
Partial work of https://github.com/biomejs/biome/issues/880 .

- translated `ja/reference/vscode.mdx`.
- fixed to settings.json instead of biome.json in `reference/vscode.mdx`.
  - let me know if you'd prefer to separate the PRs.

### CODE REVIEW

File: website/src/content/docs/reference/vscode.mdx
Comment (unvalley) [inline website/src/content/docs/reference/vscode.mdx:1]: Nice!

File: website/src/content/docs/ja/reference/vscode.mdx [hunk 1]
```diff
@@ -0,0 +1,141 @@
+---
+title: VSCode拡張機能
+description: BiomeのVSCode拡張機能について
+---
+# Biome VS Code 拡張機能
+
+[Biome](https://biomejs.dev/) は、開発スタックを統合し、複数のツールの機能を一つにまとめます。単一の設定ファイルを使用し、素晴らしいパフォーマンスで、あらゆるスタックで動作します。この拡張機能はBiomeをエディターで使えるようにし、以下のことができるようになります：
```
Comment (unvalley) [inline website/src/content/docs/ja/reference/vscode.mdx:7]: Is "環境" might be suitable?
```suggestion
[Biome](https://biomejs.dev/) は、開発スタックを統合し、複数のツールの機能を一つにまとめます。単一の設定ファイルを使用し、素晴らしいパフォーマンスで、あらゆる環境で動作します。この拡張機能はエディタ上で、以下のことを可能にします：
```

File: website/src/content/docs/ja/reference/vscode.mdx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+- ファイルを*保存時*にフォーマット、*ドキュメントのフォーマット*コマンドの使用
+- タイプ中に静的解析の結果を確認し、コード修正を適用
+- リファクタリングの実行
+
+## インストール方法
+
+VS Codeの拡張機能をインストールするには、[Visual Studio Code マーケットプレイスページ](https://marketplace.visualstudio.com/items?itemName=biomejs.biome)にアクセスするか、VS Code内で以下の方法でインストールできます：
+
+- *拡張機能* タブ（_表示_ → _拡張機能_）を開き、Biomeを検索する。
+- _クイックオープンオーバーレイ_（<kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd>+<kbd>P</kbd>または _移動_ → _ファイルに移動_）で `ext install biomejs.biome` と入力し、Enterキーを押す。
```
Comment (unvalley) [inline website/src/content/docs/ja/reference/vscode.mdx:18]: I think Command Palette is official word: https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette
```suggestion
- _Command Palette_（<kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd>+<kbd>P</kbd>または _移動_ → _ファイルに移動_）で `ext install biomejs.biome` と入力し、Enterキーを押す。
```

File: website/src/content/docs/ja/reference/vscode.mdx [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+```json title="settings.json"
+{
+	"editor.defaultFormatter": "<other formatter>",
+	"[javascript]": {
+		"editor.defaultFormatter": "biomejs.biome"
+	}
+}
+```
+
+この設定の場合、BiomeをJavaScript ファイルのデフォルトフォーマッタとして設定します。その他のファイルは`<other formatter>`を使ってフォーマットされます。
```
Comment (unvalley) [inline website/src/content/docs/ja/reference/vscode.mdx:45]:
```suggestion
この設定の場合、BiomeをJavaScriptファイルのデフォルトフォーマッタとして設定します。その他のファイルは`<other formatter>`を使ってフォーマットされます。
```

File: website/src/content/docs/ja/reference/vscode.mdx [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+
+## 設定の読み込み
+
+拡張機能は、ワークスペースのルートディレクトリから`Biome.json`ファイルを自動的に読み込みます。
```
Comment (unvalley) [inline website/src/content/docs/ja/reference/vscode.mdx:49]:
```suggestion
拡張機能は、ワークスペースのルートディレクトリから`biome.json`ファイルを自動的に読み込みます。
```

File: website/src/content/docs/ja/reference/vscode.mdx [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+
+## Biomeの読み込み
+
+拡張機能は、プロジェクトのローカル依存関係（`node_modules/Biome`）からBiomeを使用しようとします。NPMスクリプトと拡張機能が同じBiomeバージョンを使用することを確実にするために、Biomeをプロジェクトの依存関係に追加することをおすすめします。
```
Comment (unvalley) [inline website/src/content/docs/ja/reference/vscode.mdx:53]: We also should fix the original doc?
```suggestion
拡張機能は、プロジェクトのローカル依存関係（`node_modules/@biomejs/biome`）からBiomeを使用しようとします。npmスクリプトと拡張機能が同じバージョンを使用することを確実にするために、Biomeをプロジェクトの依存関係に追加することをおすすめします。
```

File: website/src/content/docs/ja/reference/vscode.mdx [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+    "quickfix.biome": true
+  }
+}
+```
+
+これは、VS Codeの`settings.json`で行います。
+
+### インポートの並び替え [実験的機能]
+
+Biomeの、VS Code拡張機能は、「Organize Imports」コードアクションを通じてインポートの並び替えをサポートしています。デフォルトでは、このアクションは<kbd title="Shift">⇧</kbd>+<kbd>Alt</kbd>+<kbd>O</kbd>のキーボードショートカットを使用して実行することができます。または、_コマンドパレット_ (<kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd>+<kbd title="Shift">⇧</kbd>+<kbd>P</kbd>) を開き、「Organize Imports」を選択してアクセスすることもできます。
```
Comment (unvalley) [inline website/src/content/docs/ja/reference/vscode.mdx:88]:
```suggestion

BiomeのVS Code拡張機能は、Organize Imports コードアクションを通じてインポートの並び替えをサポートしています。デフォルトでは、このアクションは<kbd title="Shift">⇧</kbd>+<kbd>Alt</kbd>+<kbd>O</kbd>のキーボードショートカットを使用して実行することができます。または、_コマンドパレット_ (<kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd>+<kbd title="Shift">⇧</kbd>+<kbd>P</kbd>) を開き、Organize Imports を選択してアクセスすることもできます。
```

File: website/src/content/docs/ja/reference/vscode.mdx [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+	}
+}
+```
+
+## 拡張機能の設定
+
+### `biome.lspBin`
+
+`biome.lspBin`オプションは、拡張機能が使用するBiomeのバイナリを上書きします。
+相対的なパスを指定した場合は、ワークスペースフォルダが基準パスとして使用されます。
```
Comment (unvalley) [inline website/src/content/docs/ja/reference/vscode.mdx:105]:
```suggestion
相対パスを指定した場合は、ワークスペースフォルダが基準パスとして使用されます。
```

**Review by unvalley — APPROVED** [review]
Thank you! I left some suggestions. We need to update some sentences on the original doc. If you have time, I would like to have it done:)

File: website/src/content/docs/reference/vscode.mdx
Comment (sakimyto) [reply website/src/content/docs/reference/vscode.mdx:1]: 🙌

**Comment by sakimyto** [comment]
@unvalley 
Thank you for your reviews.

> Thank you! I left some suggestions but they are nits. We need to update some sentences on the original doc. If you have time, I would like to have it done:)

Is this right?

https://github.com/biomejs/biome/pull/938/commits/93522c316dfbf5de1fe391c53c60998950b38d46

**Pull request merged**